### DISCUSSION TITLE
Better MacOS installer

### DISCUSSION THREAD

**Discussion by thewh1teagle** [tauri-apps/tauri#8691] [Ideas]
Add background image like this one
![image](https://github.com/tauri-apps/tauri/assets/61390950/b2767183-abf1-435a-b713-9ad7df5c9ad7)

**Comment by FabianLars** [comment]
This is supported in v2. A backport was started here https://github.com/tauri-apps/tauri/pull/8334 but we had no time to look at it yet.
